lib/Elastica/Bulk/ResponseSet.php

```diff
@@ -42,16 +42,13 @@ public function getBulkResponses()
      */
     public function getError()
     {
-        $error = '';
-
         foreach ($this->getBulkResponses() as $bulkResponse) {
             if ($bulkResponse->hasError()) {
-                $error = $bulkResponse->getError();
-                break;
+                return $bulkResponse->getError();
```

Comment:
I personally prefer to have only one return statement per method. I'm aware that this is a personal preference but I would like to keep it the same across the whole library.